release-team/role-handbooks/bug-triage/README.md

```diff
@@ -1,56 +1,81 @@
 
-1. Leaving a comment on the GitHub issue or PR, e.g. "This issue hasn't been updated in 3 months.  Are you still expecting to complete it for 1.11?".  It's helpful here to @ mention individuals or SIG ```-bugs``` or ```-pr-reviews``` aliases, e.g. "@kubernetes/sig-node-bugs" or "@kubernetes/sig-network-pr-reviews".
-2. Sending a message to the SIG channel or mailing list about the problem.  It's helpful here to condense multiple issues into a list, e.g. "Hey, these three issues haven't seen any attention, are they still valid for 1.11?"
-3. Messaging individual owners and reviewers via Slack and/or direct email (GitHub notification emails must have been filtered or missed if you're past step #1).  Individual's email addresses may be harder to find than GitHub ID, but ure usually in the git commit history.  Slack handles are often harder yet to find.  There is no master list mapping human names to GitHub ID, email or Slack ID.  If you can't find contact info, asking in the appropriate SIG's Slack channel will usually get you pointed in the right direction.
-4. Escalating to the Release Team Lead with suggestions on what to do with non-responsive issues.
+### Escalation Path
+Whenever you find a bug that needs to be "fixed" or kicked out of the release, try the following escalation path:
 
-In general, you should never decide whether something is in or out of a milestone; the SIG or the Release Team Lead needs to do that.  In practice, you should fix anything simple that saves folks time when the intent is obvious or a milestone decision has been made.  For example, you might add/modify kind and priority labels for a PR to match a correlating issue, or make sure an urgently awaited PR is in the milestone so it will pass CI.  See [the documentation for issue kind labels](https://git.k8s.io/community/contributors/devel/release.md#issue-kind-label)
+1. Leave a comment on the GitHub issue or PR: "This issue hasn't been updated in 3 months.  Are you still expecting to complete it for 1.14?".  It's helpful here to @ mention individuals or SIG ```-bugs``` or ```-pr-reviews``` aliases, e.g. "@kubernetes/sig-node-bugs" or "@kubernetes/sig-network-pr-reviews".
+2. Send a message to relevant SIG slack channels or mailing list about the problem: It's helpful to condense multiple issues into a list, e.g. "Hey, these three issues haven't seen any attention, are they still valid for 1.11?"
```

Comment:
Here it's useful to suggest the person post not just blanket into the channel, but explicitly @ mention the names of the issue(s) / PR(s) owners and SIG leadership.

Reply:
Calling out individuals increases the likelihood of one of them seeing it.  Or somebody else in the channel who thinks "that's not my responsibility" may be encouraged to speak up with some tangential knowledge like "hey I don't know they answer, but the person you pinged is away this week, may @otherperson can help in their absence".